minimesos/src/main/java/com/containersol/minimesos/cluster/ClusterRepository.java

```diff
@@ -22,6 +22,7 @@
     /**
      * Loads representation of the running cluster
      *
+     * @param factory TODO(mwl)
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://ci.infra.container-solutions.com/sonar/coding_rules#rule_key=squid%3AS1135)